website/content/docs/pipelines/config.mdx

```diff
@@ -0,0 +1,177 @@
+---
+layout: docs
+page_title: Pipelines Configuration
+description: |-
+  The page for detailing Waypoint pipeline configurations
+---
+
+# Pipelines Configuration
+
+This section details some of the various step built-in plugins and their
+configuration options.
+
+## Built-In Step Config Options
+
+Currently, Pipeline steps have "built-in" plugins that are backed by the existing
+Waypoint job system. Below documents each of these built-in steps and their
+parameters.
+
+### `build`
+
+This built-in step will run a Build operation job, similar to running `waypoint build`.
+
+```hcl
+step "my-build-step" {
+  use "build" {
+    disable_push = false
+  }
+}
+```
+
+##### disable_push
+
+- Type: **bool**
+- **Optional**
+
+### `deploy`
+
+This built-in step will run a Deploy operation job, similar to running `waypoint deploy`.
+
+```hcl
+step "my-deploy-step" {
+  use "deploy" {
+  }
+}
+```
+
+### `release`
+
+This built-in step will run a Release operation job, similar to running `waypoint release`.
+
+```hcl
+step "my-release-step" {
+  use "release" {
+    prune        = true
+    prune_retain = 4
+  }
+}
+```
+
+##### deployment_ref
+
+0 or unset means do a release with the latest deployment. Otherwise this accepts
+a Deployment sequence to release.
```

Comment:
What's a Deployment sequence? Is that the ID? Something else? Should we link elsewhere in our docs to tell people about how to find that?
```suggestion
0 or unset will use the latest deployment for a release. To specify a Deployment
for release, this accepts Deployment sequence numbers.
```